modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/multiple/CheckboxMultipleDDMFormFieldValueRendererTest.java

```diff
@@ -73,40 +73,7 @@ public void testRender() throws Exception {
 		ddmFormFieldOptions.addOptionLabel(
 			"value 1", LocaleUtil.US, "option 1");
 		ddmFormFieldOptions.addOptionLabel(
-			"value 2", LocaleUtil.US, "option 2");
```

Comment:
Hi @leticia-maciel, can we keep the 
```
ddmFormFieldOptions.addOptionLabel("value 2", LocaleUtil.US, "option 2");`
```
and add a third option like 

```
ddmFormFieldOptions.addOptionLabel("value 3", LocaleUtil.US, "option with &");
```
?

This way, when creating the `DDMFormFieldValue`, we can do something like:

```
new UnlocalizedValue("[\"value 1\",\"value 3"]")
```
to make sure the unselected `value 2` will not be rendered.